docs/models.py

```diff
@@ -212,6 +221,83 @@ def sync_to_db(self, decoded_documents):
             )
             document.save(update_fields=("metadata",))
 
+        self._sync_blog_to_db()
+        self._sync_views_to_db()
+
+    def _sync_blog_to_db(self):
+        """
+        Sync the blog entries into search based on the release documents
+        support end date.
+        """
+        if self.lang == "en" and self.support_end:
+            for entry in Entry.objects.published(self.support_end).searchable():
+                Document.objects.create(
+                    release=self,
+                    path=entry.get_absolute_url(),
+                    title=entry.headline,
+                    metadata={
+                        "body": entry.body_html,
+                        "breadcrumbs": [
+                            {
+                                "path": DocumentationCategory.WEBSITE.value,
+                                "title": "News",
+                            },
+                        ],
+                        "parents": DocumentationCategory.WEBSITE.value,
+                        "slug": entry.slug,
+                        "title": entry.headline,
+                        "toc": "",
+                    },
+                    config=TSEARCH_CONFIG_LANGUAGES.get(
+                        self.lang[:2], DEFAULT_TEXT_SEARCH_CONFIG
+                    ),
```

Comment:
Can we factor this into a helper function that can be reused also in `sync_views_to_db()`?

```suggestion
                    config=get_search_config(self.lang),
```